apps/src/applab/DesignToggleRow.jsx

```diff
@@ -47,9 +49,43 @@ module.exports = React.createClass({
   render: function () {
     var selectDropdown;
     var dropdownStyle = {
-      width: 140,
-      marginLeft: 10
+      display: 'inline-block',
+      verticalAlign: 'top',
+      width: 130,
+      height: 28,
+      marginBottom: 6,
+      borderColor: '#949ca2'
+    };
+
+    var buttonStyle = {
+      display: 'inline-block',
+      verticalAlign: 'top',
+      border: '1px solid #949ca2',
+      margin: '0 0 8px 0',
+      padding: '3px 6px',
+      fontSize: 14
+    };
+    var buttonPrimary = {
+      backgroundColor: '#ffa000',
+      color: '#fff'
     };
+    var buttonSecondary = {
+      backgroundColor: '#e7e8ea',
+      color: '#949ca2'
+    };
+
+    var codeButtonStyle = $.extend({}, buttonStyle, {
+      borderBottomRightRadius: 0,
+      borderTopRightRadius: 0,
+      borderRightWidth: 0
+    }, (this.state.mode === Mode.CODE ? buttonSecondary : buttonPrimary));
+
+    var designButtonStyle = $.extend({}, buttonStyle, {
+      borderBottomLeftRadius: 0,
+      borderTopLeftRadius: 0
+    }, (this.state.mode === Mode.CODE ? buttonPrimary : buttonSecondary));
```

Comment:
There are a couple of libraries out there for managing more complex inline styles in react (I think radium and react-style are the main ones). As we starting doing more complex styles, I wonder if we might want to explore these options. I personally haven't looked at them in much detail of yet, so I'm not sure how much they would get us.